spec/controllers/admin/contents_controller_spec.rb

```diff
@@ -61,11 +60,17 @@ module Alchemy
 
     describe "#order" do
       context "with content_ids in params" do
+        let(:element) do
+          create(:element, name: 'all_you_can_eat', create_contents_after_create: true)
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.